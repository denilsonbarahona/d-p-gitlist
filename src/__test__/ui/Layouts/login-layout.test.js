import React from 'react';
import { mount } from 'enzyme';
import {create} from 'react-test-renderer'
import LoginLayout from '../../../entities/login/infrastructure/layouts/login-layout';


describe('Login Layout', ()=>{
    const layout = mount(<LoginLayout />)
    test('render layout',()=>{
        expect(layout.length).toEqual(1)
    })

    test('create snapshot',()=>{
        const layout = create(<LoginLayout />)
        expect(layout.toJSON()).toMatchSnapshot()
    })
})

