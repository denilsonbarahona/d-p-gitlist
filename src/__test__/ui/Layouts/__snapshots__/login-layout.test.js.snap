// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Layout create snapshot 1`] = `
<div
  className="login"
/>
`;
